Core/include/Acts/Surfaces/ConeSurface.hpp

```diff
@@ -199,6 +199,18 @@ class ConeSurface : public Surface {
   /// Return properly formatted class name for screen output
   std::string name() const override;
 
+  /// Calculate the derivative of path length at the geometry constraint or POCA
```

Comment:
```suggestion
  /// Calculate the derivative of path length at the geometry constraint or point-of-closest-approach
```
Suggestion: more clarity for the uninitiated.